### ISSUE TITLE
Question regarding learning rate and absent sound

### ISSUE THREAD

**Issue by gancx** [haidog-yaqub/DPMTSE#3]
Hi,

Thanks for your work and effort. I have two questions about the paper. 
1. I wonder if the learning rate is fixed during the training. 
2. Have you ever investigated the case of the absence of target sound, which means the target sound does not exist in the mixture?

**Comment by haidog-yaqub** [comment]
Thanks for your interest!

1. Yes, in this work we used a fixed learning rate.
2. We did not include samples in which the target sound was totally absent.

If you are exploring target sound extraction, please check out our latest work, SoloAudio.
